### ISSUE TITLE
Detail View Icon

### ISSUE THREAD

**Issue by adrianoforster** [IvanJosipovic/BlazorTable#253]
Its possible to only show de detail icon if a row has any detail to show?

Exemple:

If row 2, dosent has a data to show in detail, hide the detail icon on parent table

![image](https://user-images.githubusercontent.com/47247301/106903279-942edb80-66d8-11eb-8dcc-375a6cec54c5.png)

**Comment by irushabh** [comment]
Yes, You have to add one condition on Table component if details template is not null then show the icon else hide the icon.
